mesonbuild/backend/ninjabackend.py

```diff
@@ -1899,6 +1908,10 @@ def generate_single_compile(self, target, outfile, src, is_generated=False, head
                 abs_src = src.fname
             else:
                 abs_src = os.path.join(self.environment.get_build_dir(), src.fname)
+        elif isinstance(src, mesonlib.File):
```

Comment:
It breaks if this is not there. Let's fix it when ripping out RawFile. Weirdly, I thought I already did it ages ago but apparently not.